lib/redix/start_options.ex

```diff
@@ -122,6 +122,9 @@ defmodule Redix.StartOptions do
             raise ArgumentError,
                   "when using Unix domain sockets, the port must be 0, got: #{inspect(port)}"
 
+          {{:local, _unix_socket_path} = host, {:ok, 0}} ->
+            {host, 0}
+
```

Comment:
This looks great! Good catch. Can we move this `case` clause up _before_ the `port != 0` one, so that it reads slightly more natural? When that's done, we're good to to!

Reply:
Awesome, thanks for this quick feedback, appreciate it!

Pushing a change now 🙇

Reply:
In fact, with this, we could remove the `when port != 0` check, what do you think?

Reply:
@mourjo yes absolutely, the second claiuse can be `{:ok, _non_zero_port}` :)

Reply:
Awesome, did this as well (except the leading `_` since the error message uses the port information